app/routers/dayview.py

```diff
@@ -0,0 +1,112 @@
+from datetime import datetime, timedelta
+from typing import Optional
+
+from fastapi import APIRouter, Depends, Request
+from fastapi.templating import Jinja2Templates
+from sqlalchemy import and_, or_
+
+from app.database.database import get_db
+from app.database.models import Event, User
+from app.dependencies import TEMPLATES_PATH
+
+
+templates = Jinja2Templates(directory=TEMPLATES_PATH)
+
+
+router = APIRouter()
+
+
+class DivAttributes:
+    GRID_BAR_QUARTER = 1
+    FULL_GRID_BAR = 4
+    MIN_MINUTES = 0
+    MAX_MINUTES = 15
+    BASE_GRID_BAR = 5
+    FIRST_GRID_BAR = 1
+    LAST_GRID_BAR = 101
+
+    def __init__(self, event: Event, day: Optional[datetime] = False) -> None:
+        self.start_time = event.start
+        self.end_time = event.end
+        self.day = day
+        self._check_if_multiday_event()
```

Comment:
Prefer to return a value and set it here as a variable, if possible.

It makes the function testable and more reusable.